### PR TITLE
Faster way of counting spikes

### PR DESCRIPTION
**Pull request by dimokaramanlis**

### CODE REVIEW

**Comment by marius10p** [comment]
Thanks, give me a few days until I can test it.

**Comment by alexmorley** [comment]
LGTM. The following runs fine:

```
Nk = 10;
st3 = randsample(1:Nk, 1000, true);

nspk_1 = accumarray(st3(:), 1, [Nk, 1], @sum);


nspk_2 = zeros(Nk, 1);
for j = 1:Nk
    nspk_2(j) = sum(st3==j); % determine total number of spikes in each neuron
end

assert(all(nspk_1 == nspk_2))
```

File: postProcess/find_merges.m [hunk 1]
```diff
@@ -10,10 +10,7 @@
 Xsim = Xsim - diag(diag(Xsim)); % remove the diagonal of ones
 
 % sort by firing rate first
-nspk = zeros(Nk, 1);
-for j = 1:Nk
-    nspk(j) = sum(rez.st3(:,2)==j); % determine total number of spikes in each neuron
-end
+nspk = accumarray(rez.st3(:,2), 1, [Nk, 1], @sum);
```
Comment (alexmorley) [inline postProcess/find_merges.m:13]:
```suggestion
nspk = accumarray(rez.st3(:,2), 1, [Nk, 1], @sum); % determine total number of spikes in each neuron
```

Maybe let's keep the comment in? :)

Comment (dimokaramanlis) [reply postProcess/find_merges.m:13]: Sure, I missed that

**Pull request merged**